### PR TITLE
[RHELC-1037] Auto generate manpage Github action fix

### PR DESCRIPTION
**Pull request by Andrew-ang9**
The auto-gen manpage PR was believed to be working but when it was time to make the manpage for the release it wasn't correct and so it was found out that it wasn't running correctly. 

Jira Issues: [RHELC-1037](https://issues.redhat.com/browse/RHELC-1037)

Checklist

- [ ] PR has been tested manually in a VM (either author or reviewer)
- [ ] Jira issue has been made public if possible
- [ ] `[RHELC-]` is part of the PR title 
- [ ] GitHub label has been added to help with Release notes 
- [ ] PR title explains the change from the user's point of view
- [ ] Code and tests are documented properly
- [ ] The commits are squashed to as few commits as possible (without losing data) 
- [ ] When merged: Jira issue has been updated to `Release Pending` if relevant

### CODE REVIEW

File: .github/workflows/generate_manpage.yml [hunk 1]
```diff
@@ -31,3 +31,17 @@ jobs:
         run: |
           chmod +x scripts/manpage_generation.sh
           bash scripts/manpage_generation.sh
+
+      - name: Show File Modification
+        run: |
+          FILE_PATH="man/convert2rhel.8"
+
+          # Fetch the changes from the remote repository
+          git fetch --quite
```
Comment (SpyTec) [inline .github/workflows/generate_manpage.yml:40]:
```suggestion
          git fetch --quiet
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ jobs:
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
 
       - name: Trigger Manpage Generation
-        uses: ./.github/workflows
```
Comment (SpyTec) [inline .github/workflows/release.yml:17]: Did you check this in your repo? Do we know if this works?

Comment (SpyTec) [reply .github/workflows/release.yml:17]: Looking at the docs it seems like we're using it incorrectly still

https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow

Comment (Andrew-ang9) [reply .github/workflows/release.yml:17]: ohh I didn't check to see if it worked in my repo first. I do know that the last commit that I did was passing on my repo

Comment (SpyTec) [reply .github/workflows/release.yml:17]: To test this the easiest way would be to do it in your PR with the trigger for `release.yml` changed to 

```
on:
  push
```

File: .github/workflows/generate_manpage.yml (hunk 1 above)
Comment (Andrew-ang9) [reply .github/workflows/generate_manpage.yml]: Done!

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -21,13 +21,7 @@ jobs:
...
 
       - name: Trigger Manpage Generation
-        run: |
-          curl -X POST \
-            -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
-            -H "Accept: application/vnd.github.everest-preview+json" \
-            -H "Content-Type: application/json" \
-            https://api.github.com/repos/${{ github.repository }}/dispatches \
-            --data '{"event_type": "trigger-manpage-generation"}'
+        uses: .github/workflows/generate_manpage.yml
```
[the remark this answers is not in the document]
Comment (SpyTec) [reply .github/workflows/release.yml]: Though didn't you try `./.github/workflows/generate_manpage.yml` before and that didn't work?

[the remark this answers is not in the document]
Comment (Andrew-ang9) [reply .github/workflows/release.yml]: Yeah I did try ./.github/workflows/generate_manpage.yml and it didn't work but i'll give it a shot anyways. Thank you for the suggestion

**Comment by SpyTec** [comment]
Michal mentioned something we didn't think of, that the downstream release is actually done through the source code tarball that's in the GitHub release assets page. Since we cannot and shouldn't modify this ourselves, just changing the manpage on the main branch would be a better solution as we then cover all bases with the source code tarball and release rpms

Closing for that solution and will update the Jira issue

**Pull request closed without merging**